src/tokenizer/CMakeLists.txt

```diff
@@ -8,11 +8,10 @@ file(GLOB tokenizer_srcs CONFIGURE_DEPENDS
 )
 
 FetchContent_Declare(GSL URL https://github.com/microsoft/GSL/archive/refs/tags/v4.0.0.zip)
-FetchContent_MakeAvailable(GSL)
+onnxruntime_fetchcontent_makeavailable(GSL)
```

Comment:
Move the URLs to cmake/deps.txt and run the script under https://github.com/microsoft/onnxruntime-genai/tree/main/cgmanifests , so that they will be registered to CG.